### PR TITLE
UI - Document Upload - Align with Design

### PR DESCRIPTION
**Pull request by shaangill025**
*Issue:*

- https://github.com/bcgov/entity/issues/22520

*Description of changes:*
 - [x] Clear document from the document picker after uploading it
 - [x] Display the full file name regardless of the length of the file name
 
Design:
image

With this PR:
image
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-124-sn5o2url.web.app

File: strr-web/components/bcros/form-section/principal-residence/Form.vue [hunk 1]
```diff
@@ -226,10 +253,48 @@ const otherExemptionReasons: string[] = [
   tPrincipalResidence('strataGuest')
 ]
 
+const formatFileName = (fileName: string) => {
+  if (fileName.length <= 25) {
+    return fileName
+  }
+  const extension = fileName.split('.').pop()
+  const nameWithoutExtension = fileName.substring(0, fileName.lastIndexOf('.'))
+  return `${nameWithoutExtension.substring(0, 21)}....${extension}`
```
[the remark this answers is not in the document]
Comment (shaangill025) [reply strr-web/components/bcros/form-section/principal-residence/Form.vue:262]: This is from the design, I think I need to remove the extension.
image

[the remark this answers is not in the document]
Comment (shaangill025) [reply strr-web/components/bcros/form-section/principal-residence/Form.vue]: @dimak1 I have made changes to the `formatFileName` function to align more with the above mobile design. The screenshots in the description are now updated.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-124-sn5o2url.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-124-sn5o2url.web.app

File: strr-web/components/bcros/form-section/principal-residence/Form.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (shaangill025) [reply strr-web/components/bcros/form-section/principal-residence/Form.vue]: https://github.com/bcgov/STRR/blob/a509223a021c5358bf3da78d031bfa915b2fa8b8/strr-web/components/bcros/form-section/principal-residence/Form.vue#L130

Implemented alignment with mobile design with CSS.

**Review by dimak1 — APPROVED** [review]
Nice, much cleaner, great job!

**Pull request merged**